modules/health_quest/app/services/health_quest/user_service.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+Faraday::Middleware.register_middleware health_quest_logging: HealthQuest::Middleware::HealthQuestLogging
+
+module HealthQuest
+  class UserService < HealthQuest::BaseService
```

Comment:
I don't see the UserService being used anywhere.  Is this what this PR is adding?

Reply:
Yes, the whole set of changes would be 2084 lines or so, but I cut it down to 800+ lines and now I have another PR that is around 325 lines so I have to remove some things